apps/sms/test/marionette/inbox_test.js

```diff
@@ -0,0 +1,79 @@
+/* global require, marionette, setup, suite, test, __dirname */
```

Comment:
This test doesn't really test the issue we're fixing as it's not possible currently due to marionette apps plugin limitation (app.launch waits for the "load" event), so it just tests that we can enter new message view while rendering is still in progress. Let me know if you prefer to not include this test in this patch :)